packages/dialtone-vue3/components/tabs/tab_group.vue

```diff
@@ -171,6 +172,7 @@ export default {
 
   mounted () {
     this.updateSelected();
+    warnIfUnmounted(this.$el, this.$options.name);
```

Comment:
Does this one actually need it? What renders at the body here?

Reply:
Nothing here, it's needed in the tests because keyboard events are tested and for those to work correctly the component needs to be rendering in the real DOM.
Probably the warning here is not necessary then.